pkg/envoy/resources.go

```diff
@@ -123,6 +132,10 @@ func (cache *NPHDSCache) handleIPDelete(npHost *envoyAPI.NetworkPolicyHosts, pee
 		} else {
 			npHost.HostAddresses = append(npHost.HostAddresses[0:targetIndex], npHost.HostAddresses[targetIndex+1:]...)
 		}
+		if err := npHost.Validate(); err != nil {
+			scopedLog.WithError(err).Warning("Could not validate NPHDS delete message")
```

Comment:
NPHDS delete message -> NPHDS resource after deleting host

Reply:
NPHDS implies host at the moment, and I'll need to update the "host" part almost immediately with my new CIDR PR. I can 's/message/resource/' though

Reply:
👍🏻